ios/Classes/MobileScanner.swift

```diff
@@ -48,6 +48,10 @@ public class MobileScanner: NSObject, AVCaptureVideoDataOutputSampleBufferDelega
 
     var detectionSpeed: DetectionSpeed = DetectionSpeed.noDuplicates
 
+    /// Analyze inverted image intervally to include both inverted and normal images
```

Comment:
```suggestion
var intervalInvertImage: Bool = false
private var invertCurrentImage: Bool = false
```

Same name suggestion as on Android. The comments can also be removed.

Reply:
I kept the private temp var's comment and enhanced it. Let me know if that's alright
![image](https://github.com/user-attachments/assets/115afdd3-7dd4-45af-abf0-642684060d07)